README.md

```diff
@@ -1,2 +1,8 @@
 # iterativereview
 This repo is to test out Github's iterative review experience. Yay!
+
+This is a line I added.
```

Comment:
(Comment on a filtered diff view) (Line retained)

Can you add why you added this line?